bin/Debugger.Sample/Debugger.Sample.cpp

```diff
@@ -423,6 +498,11 @@ int _cdecl wmain(int argc, wchar_t* argv[])
         // Now set the execution context as being the current one on this thread.
         IfFailError(JsSetCurrentContext(context), L"failed to set current context.");
 
+        if (arguments.enableConsoleRedirect)
+        {
+            IfFailError(RedirectConsoleToDebugger(debugProtocolHandler.get()), L"Failed to redirect to debugger's console");
```

Comment:
Suggest not using possessive "debugger's". Suggest "Failed to redirect console to debugger."